src/renderer/Container/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import configuration from '~/src/configuration';
+import { Container as ContainerInterface } from '~/src/models/Container/types';
+import { useSelector } from '~/src/store/hooks';
+import { selectPlayerPosition } from '~/src/store/reducers/player/selectors';
+import positionsEqual from '~/src/utilities/positionsEqual';
+
+import './container.css';
+import ContainerOverlay from './ContainerOverlay';
+
+interface ContainerProps {
+  container: ContainerInterface;
+}
+
+const Container: React.FC<ContainerProps> = ({ container }) => {
+  const playerPosition = useSelector(selectPlayerPosition);
+
+  return (
+    <React.Fragment>
+      <div
+        className={`container container--${container.variant}`}
+        style={{
+          left: container.position.left * configuration.tileSize(),
+          top: container.position.top * configuration.tileSize(),
+        }}
+      />
+      {positionsEqual(playerPosition, container.position) && (
```

Comment:
This check is not required, it is also checked inside the overlay which is better.